src/core/AutomationClip.cpp

```diff
@@ -489,12 +530,31 @@ TimePos AutomationClip::setDragValue(
 
 	generateTangents();
 
+	TimePos returnedPos;
+
 	if (m_dragKeepOutValue)
 	{
-		return this->putValues(time, value, m_dragOutValue, quantPos, controlKey);
+		returnedPos = this->putValues(time, value, m_dragOutValue, quantPos, controlKey);
+	}
+	else
+	{
+		returnedPos = this->putValue(time, value, quantPos, controlKey);
```

Comment:
The ``outValue`` parameter for this function would now be the argument ``quantPos``. Is this intended?

Reply:
`AutomationClip::putValue` takes a single value and uses it for both the inValue and outValue and `AutomationClip::putValues` takes a value for each of them

Reply:
My bad, didn't see that it was a completely different function.